src/main/java/com/shippo/model/LabelFileType.java

```diff
@@ -0,0 +1,5 @@
+package com.shippo.model;
+
+public enum LabelFileType {
+	PNG, PDF, PDF_4X6, ZPLII
```

Comment:
I think 'PDF_4X6' needs to be 'PDF_4x6' (notice the lower case 'x'), else it would return an error. I ran into this with the C# library. Please test to make sure.